web/src/Components/Bridging/Graphs/BridgeGraph.tsx

```diff
@@ -9,11 +19,11 @@ type BridgeGraphProps = {
   title: string
   graphData: GraphData[]
   sizeFractions: number[]
-  bridges: Bridge | undefined
+  bridges: Bridge
   showBridge?: boolean
 }
 
-const CustomTooltip = ({ active, payload, label }) => {
+const CustomTooltip = ({ active, payload, label }: TooltipProps<number, string>): ReactElement => {
```

Comment:
```suggestion
const CustomTooltip = ({ active, payload, label }: TooltipProps<number, string>): ReactElement | false => {
```

Instead of returning an empty tag we want to return false or null, I would go for false